### PR TITLE
Added OnClick and ClickedCommand to Xamarin.Forms.iOS/Droid/UWP

### PR DESCRIPTION
**Pull request by RustamIrzaev**
Hello.

I added `OnClick` event and `ClickedCommand` command to Xamarin.Forms.iOS project. 
Also, I made changes to an example project to reflect my changes.

Thank you in advance.

### CODE REVIEW

File: Lottie.Forms.iOS/Renderers/AnimationViewRenderer.cs [hunk 1]
```diff
@@ -99,6 +99,8 @@ private void InitAnimationViewForElement(AnimationView theElement)
                 AnimationSpeed = theElement.Speed
             };
 
+            _animationView.AddGestureRecognizer(new UITapGestureRecognizer(theElement.Click));
```
Comment (modplug) [inline Lottie.Forms.iOS/Renderers/AnimationViewRenderer.cs]: This needs to be unsubcribed to  upon disposing the old _animationView

**Comment by RustamIrzaev** [comment]
I added unsubscribe logic to Forms.iOS and implementation to Forms.Droid and Forms.UWP.

Hopefully made this well.
It would be nice to hear what's wrong @modplug 

Thanks.

**Comment by modplug** [comment]
Hi @RustamIrzaev Just had a look at the UWP and Android implementations and they look good. One final question before we merge is why it is necessary to have seperate implementations for OnClick. Couldn't this be solved in a generic way by adding a TapGestureRecognizer in the Xamarin Forms implementation of AnimationView?

**Comment by RustamIrzaev** [comment]
If I do something like this in `Lottie.Forms.AnimationView` then nothing works

```csharp
var gestureRecognizer = new TapGestureRecognizer();
gestureRecognizer.Tapped += (sender, e) => Click();
GestureRecognizers.Add(gestureRecognizer);
```

**Comment by modplug** [comment]
I see. That is probably because we switch out the underlying view when changing animations. Thanks for the contribution, please keep them coming in the future 👍

**Pull request merged**